package.json

```diff
@@ -2,9 +2,9 @@
     "name": "dynamites-ecomm-be",
     "version": "1.0.0",
     "description": "",
-    "main": "index.js",
+    "main": "src/index.ts",
     "scripts": {
-        "start": "node ./dist/index.js",
+        "start": "nodemon --watch",
```

Comment:
Can we stick to node for production use

Reply:
i think this directory and  file does not exist /dist/index.js